charts/incubator/oscam/SCALE/questions.yaml

```
@@ -0,0 +1,1155 @@
+                  attrs:
+                    - variable: name
+                      label: "Name"
+                      schema:
+                        type: string
+                    - variable: value
+                      label: "Value"
+                      schema:
+                        type: string
+        - variable: annotationsList
+          label: " Controller Annotations"
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: annotationItem
+                label: "Label"
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: name
+                      label: "Name"
+                      schema:
+                        type: string
+                    - variable: value
+                      label: "Value"
+                      schema:
+                        type: string
+
+  - variable: podLabelsList
+    group: "Container Configuration"
+    label: "Pod Labels"
+    schema:
+      type: list
+      default: []
+      items:
+        - variable: podLabelItem
+          label: "Label"
+          schema:
+            type: dict
+            attrs:
+              - variable: name
+                label: "Name"
+                schema:
+                  type: string
+              - variable: value
+                label: "Value"
+                schema:
+                  type: string
+
+  - variable: podAnnotationsList
+    group: "Container Configuration"
+    label: "Pod Annotations"
+    schema:
+      type: list
+      default: []
+      items:
+        - variable: podAnnotationItem
+          label: "Label"
+          schema:
+            type: dict
+            attrs:
+              - variable: name
+                label: "Name"
+                schema:
+                  type: string
+              - variable: value
+                label: "Value"
+                schema:
+                  type: string
+
+  - variable: termination
+    group: "Container Configuration"
+    label: "Termination settings"
+    schema:
+      type: dict
+      attrs:
+        - variable: gracePeriodSeconds
+          label: "Grace Period Seconds"
+          schema:
+            type: int
+            default: 10
+  - variable: env
+    group: "Container Configuration"
+    label: "Image Environment"
+    schema:
+      type: dict
+      attrs:
+        - variable: TZ
+          label: "Timezone"
+          schema:
+            type: string
+            default: "Etc/UTC"
+            $ref:
+              - "definitions/timezone"
+        - variable: PUID
+          label: "PUID"
+          description: "Sets the userID inside the container"
+          schema:
+            type: string
+            default: "1000"
+        - variable: PGID
+          label: "PGID"
+          description: "Sets the GroupID inside the container"
+          schema:
+            type: string
+            default: "20"
```

It would be prefered if you use the Apps group and user here...
```suggestion
        - variable: PUID
          label: "PUID"
          description: "Sets the userID inside the container"
          schema:
            type: string
            default: "568"
        - variable: PGID
          label: "PGID"
          description: "Sets the GroupID inside the container"
          schema:
            type: string
            default: "568"
```

---

Merged this myself to force a test run to verify.
Should work (as it's an LSIO container) but need to verify...